docs/projects/ad4110/index.rst

```diff
@@ -0,0 +1,211 @@
+system module power supply.
+
+The :adi:`AD4110` provides internal front-end diagnostic functions to indicate
+overvoltage, undervoltage, open wire, overcurrent and overtemperature
+conditions. The high voltage input is overcurrent limited and overvoltage
+protected up to ±35 V.
+
+The :adi:`AD4110` incorporates a 24-bit sigma delta analog to digital converter
+offering conversion rates from 5 SPS to 125 kSPS with simultaneous 50 Hz and
+60 Hz noise rejection.
+
+Applications:
+
+  * Process control and industrial automation
+  * Remote and distributed control systems
+  * Instrumentation and measurement
+  * Sensor and data acquisition
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD4110-1SDZ <EVAL-AD4110-1SDZ>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD4110`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+-  :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on PMOD
+
+Block design
+-------------------------------------------------------------------------------
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The data path and clock domains are depicted in the below diagram:
+
+.. image:: ad4110_block_diagram.svg
+   :width: 800
+   :align: center
+   :alt: AD4110_SDZ block diagram
+
+
+Jumper setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+================== ================ =============================================
+Jumper/Solder link Default Position Description
+================== ================ =============================================
+SL1                A                Connects the 5 V output of the :adi:`ADP7102`
+                                    to the DVDD rail to supply the IOVDD pins of
+                                    the :adi:`AD4110-1`
+SL2                A                Valid SDP connection to evaluation board LED
+SL3                A                Connects J8 AINCOM to Pin 26 (AINCOM(LV)) of
+                                    the :adi:`AD4110-1`
+SL4                A                Connects GND to Pin 28 (AIN2(LV)) of the
+                                    :adi:`AD4110-1`
+SL5                A                Connects VOUT of the TMP36 temperature sensor
+                                    to Pin 27 (AIN1(LV)) of the :adi:`AD4110-1`
+================== ================ =============================================
+
+Hardware setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+============ ================ ==============
+Signal       AD4110 Testpoint  ZedBoard PMOD
+============ ================ ==============
+CS           CS               JA1
+DIN(MOSI)    DIN              JA2
+DOUT/RDY     DOUT             JA3
```

Comment:
This signal is unused by the HDL?
[JA3 is Zynq EPP pin Y19](https://digilent.com/reference/_media/zedboard:zedboard_ug.pdf_) not present on the [system_constr.xdc](https://github.com/analogdevicesinc/hdl/blob/main/projects/ad4110/zed/system_constr.xdc)
Also, what about pmod_gpio[3:1] and MISO?

Reply:
Regarding the JA3 pin: 
https://github.com/Digilent/digilent-xdc/blob/7583b4e7a6b8178afdb7f9e07f18162c48a89b0d/Zedboard-Master.xdc#L89
https://github.com/Digilent/digilent-xdc/blob/7583b4e7a6b8178afdb7f9e07f18162c48a89b0d/Zedboard-Master.xdc#L200
MISO is tied to DOUT/RDY.
We know nothing about pmod_gpio[3:1].